### DISCUSSION TITLE
who is using the Gateway API

### DISCUSSION THREAD

**Discussion by ZhangSetSail** [kubernetes-sigs/gateway-api#1948] [Q&A]
I want to write an article on interpreting the Gateway API, and I want to know which manufacturers are already using the Gateway API.

**Comment by shaneutt** [comment]
Hi @ZhangSetSail! 

You can find a list of known implementations on our website:

https://gateway-api.sigs.k8s.io/implementations/

Let us know if this is helpful!

**Comment by ZhangSetSail** [comment]
sorry, I may not have expressed clearly, I would like to ask which manufacturers are using Gateway API instead of Ingress in production

**Reply by shaneutt** [reply]
Right now I think that's a hard question to answer. You may want to join the `#sig-network-gateway-api` channel in https://kubernetes.slack.com and do some asking for volunteers who want to be mentioned in your article.
